diff -ruN a/Makefile b/Makefile
--- a/Makefile	2021-07-11 21:01:13.019444300 +0800
+++ b/Makefile	2021-08-15 11:41:26.000000000 +0800
@@ -7,11 +7,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mtools
-PKG_VERSION:=4.0.24
+PKG_VERSION:=4.0.35
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=@GNU/$(PKG_NAME)
-PKG_HASH:=24f4a2da9219f98498eb1b340cd96db7ef9b684c067d1bdeb6e85efdd13b2fb9
+PKG_HASH:=skip
 
 HOST_BUILD_PARALLEL:=1
 
